### PR TITLE
isClickable handles child elements

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

is clickable should return true if elementFromPoint is child of target element

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by igerasimov23** [comment]
Thank you for quick fix. I executed one test and looks fine to me!

File: packages/webdriverio/tests/scripts/isElementClickable.test.js [hunk 1]
```diff
@@ -23,6 +23,22 @@ describe('isElementClickable script', () => {
         expect(isElementClickable(elemMock)).toBe(true)
```
Comment (christian-bromann) [inline packages/webdriverio/tests/scripts/isElementClickable.test.js]: This test fails

Comment (mgrybyk) [reply packages/webdriverio/tests/scripts/isElementClickable.test.js]: Gonna fix soon

Comment (mgrybyk) [reply packages/webdriverio/tests/scripts/isElementClickable.test.js]: fixed

**Pull request merged**